Core.ApplicationServices/UserService.cs

```diff
@@ -54,7 +59,9 @@ public class UserService : IUserService
             IAuthorizationContext authorizationContext,
             IDomainEvents domainEvents,
             IUserRepository repository,
-            IOrganizationService organizationService)
+            IOrganizationService organizationService,
+            ITransactionManager transactionManager, 
+            Maybe<ActiveUserIdContext> activeUserIdContext)
```

Comment:
you could have  also used the IOrganizationalUserContext. That one will not return a maybe but always a true int